pkg/destinations/duckdb/insert.go

```diff
@@ -84,23 +84,12 @@ func (s *DuckDBServer) InsertBatchFromNDJson(table string, input io.ReadSeeker)
 		return err
 	}
 
-	connector, err := s.getConnector()
+	err = s.createTable(table, s.db)
 	if err != nil {
 		return err
 	}
 
-	db := sql.OpenDB(connector)
-	if err != nil {
-		return err
-	}
-	defer db.Close()
-
-	err = s.createTable(table, db)
-	if err != nil {
-		return err
-	}
-
-	err = s.createColumns(table, jsonTypes, db)
+	err = s.createColumns(table, jsonTypes, s.db)
```

Comment:
In practice we probably don't need to pass `s.db` in, right?

Reply:
nope